app/assets/stylesheets/components/_modal.scss

```diff
@@ -105,3 +81,11 @@
     border-color: $yellow;
   }
 }
+
+// ===============================================
+// Pending upstream Login Design System revisions:
```

Comment:
Typically I'd like to immediately include a link to a corresponding pull request in the design system which includes the specific changes we're trying to incorporate into a project ([example](https://github.com/18F/identity-idp/blob/b4f0a189c07305dfe2b6e2243b215c540f19f7b3/app/assets/stylesheets/components/all.scss#L28-L36)). Because we're not really at the point of being able to implement stylings for the modal as a whole, the intent of this comment is to signal that this code should not plan to exist here long-term, but there is no corresponding upstream changes quite yet.